services/console-proxy/server/src/com/cloud/consoleproxy/vnc/VncClient.java

```diff
@@ -16,6 +16,16 @@
 // under the License.
 package com.cloud.consoleproxy.vnc;
 
+import com.cloud.consoleproxy.ConsoleProxyClientListener;
```

Comment:
Nothing changed in this file, just rearranged a bunch of stuff. Can you revert this file back, we don't need it.